metrics/integration.go

```diff
@@ -30,11 +30,12 @@ var (
 		},
 	)
 
-	SnapshotConcurrentTotal = prometheus.NewGauge(
+	SnapshotConcurrentTotal = prometheus.NewGaugeVec(
```

Comment:
IF we change scalar to vector, this may nee follow-up story to upgdate grafana dashboard.

Reply:
Why updated grafana dashboard?  Release team has already began to use it for some time.

Reply:
I saw the "Upgrade" grafana dashboard, so I thought it's to upgrade granfana. I didn't noticed you meant to update the data represent.  It's misunderstanding.